### ISSUE TITLE
Dashboard using Orchard Framework

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-core#74]
The Dashboard area is currently not extensible, but making it so would allow for some seriously cool scenarios. For example, someone could implement application specific functionality to manage data that the workflows should work with. Or perhaps some general purpose Forms module allowing an admin to define workflow driven forms.

Many of these things work today in Orchard Core, and we should take advantage of that.

One option could be to build the dashboard as Orchard Core modules. 

Another option could be to create the Dashboard as a custom theme, taking full control over the look and feel, while still benefitting from the modular framework.

**Comment by MiroslavMikus** [comment]
Hey, very cool idea! 👍 

I see there is already [some](https://github.com/elsa-workflows/elsa-core/search?q=orchard&type=issues) *thinking* about this feature. 

Are there any plans for this feature. I could check if I can make the Orchard Core module. 
> Warning: Guidance will be required -_-

**Comment by sfmskywalker** [comment]
Absolutely. In fact, these plans are [already work in progress](https://github.com/elsa-workflows/elsa-orchard-core).

As you can see, it's been more than a couple of months since I worked on it due to shifted short-term priorities. So if you are interested you would be more than welcome to continue the work!

Right now the module is quite simple: all it does is allow the user to define a workflow server (a name and a URL pointing to some Elsa Server host).

Then it basically embeds the Elsa dashboard client components that point to a given workflow server.

Here's what I think should be done next:

- Instead of rendering the full Elsa Dashboard, we only render the workflow designer.
- The module should provide its own list view for workflow definitions, workflow instances and workflow blueprints.
- The module itself should install Elsa too, so that we can execute workflows with Orchard-specific activities in-process.
   - This means that there will always be an "elsa server" available as a feature that represents the local Orchard instance.

If you want to discuss any of this, please feel free to hit me up on Discord as you like.

**Comment by adamradocz** [comment]
Hi @sfmskywalker,

What is the state of this issue?
Could you provide any guidance on how to approach the following point:

> * Instead of rendering the full Elsa Dashboard, we only render the workflow designer.

**Comment by sfmskywalker** [comment]
An Orchard Core based dashboard no longer makes sense for Elsa3, but it received dashboarding capabilities nonetheless: https://www.elsaworkflows.io/blog/the-elsa-studio-dashboard-is-split-on-purpose
